content/tutorials/webrtc/datachannels/en/index.html

```diff
@@ -57,93 +57,92 @@
 </th>
 <tr>
-  <td style="font-weight: bold;">Reliability</td><td>reliable</td><td>unreliable</td><td>configurable</td>
+  <td style="font-weight: bold;">Reliability</td><td>Reliable</td><td>Unreliable</td><td>Configurable</td>
 </tr>
 <tr>
-  <td style="font-weight: bold;">Delivery</td><td>ordered</td><td>unordered</td><td>configurable</td>
+  <td style="font-weight: bold;">Delivery</td><td>Ordered</td><td>Unordered</td><td>Configurable</td>
 </tr>
 <tr>
-  <td style="font-weight: bold;">Transmission</td><td>byte-oriented</td><td>message-oriented</td><td>message-oriented</td>
+  <td style="font-weight: bold;">Transmission</td><td>Byte-oriented</td><td>Message-oriented</td><td>Message-oriented</td>
 </tr>
 <tr>
-  <td style="font-weight: bold;">Flow control</td><td>yes</td><td>no</td><td>yes</td>
+  <td style="font-weight: bold;">Flow control</td><td>Yes</td><td>No</td><td>Yes</td>
 </tr>
 <tr>
-  <td style="font-weight: bold;">Congestion control</td><td>yes</td><td>no</td><td>yes</td>
+  <td style="font-weight: bold;">Congestion control</td><td>Yes</td><td>No</td><td>Yes</td>
 </tr>
 </table>
 
-<p>(From <a href="http://chimera.labs.oreilly.com/books/1230000000545/ch18.html">High Performance Browser Networking</a> by <a href="http://www.igvita.com/" title="Ilya Grigorik's website">Ilya Grigorik</a>.)</p>
-
-<p>RTCDataChannel can work in unreliable and unordered mode (analogous to User Datagram Protocol or UDP), reliable and ordered mode (analogous to Transmission Control Protocol or TCP) and partial reliable modes:</p>
-
-<ul>
-  <li><strong>Reliable and ordered mode</strong> guarantees the transmission of messages and also the order in which they are delivered. This takes extra overhead, thus potentially making this mode slower. </li>
-  <li><strong>Unreliable and unordered mode</strong> does not guarantee every message will get to the other side nor what order they get there. This removes the overhead, allowing this mode to work much faster.</li>
-  <li><strong>Partial reliable mode</strong> guarantees the transmission of message under a specific condition (such as a retransmit timeout or a maximum amount of retransmissions). The ordering of messages is also configurable.
-</ul>
-
-<p>Performance for the first two modes is about the same when there are no packet losses. However, in reliable and ordered mode a lost packet will cause other packets to get blocked behind it, and the lost packet might be stale by the time it is retransmitted and arrives. It is, of course, possible to use multiple data channels within the same application, each with their own (un)reliable semantics.</p>
-
-<p>We show below how to configure RTCDataChannel to use reliable and ordered or unreliable and unordered mode.</p>
+<p>Next, you learn how to configure <code>RTCDataChannel</code> to use reliable and ordered or unreliable and unordered mode.</p>
```

Comment:
The table is adapted from Ilya's book (i.e. needs the credit).

Reply:
The attribution is still there, just before the table.